src/Theme/ThemeTypes.ts

```diff
@@ -0,0 +1,58 @@
+export type ColorRange = {
+  light: string;
+  medium: string;
+  subtle?: string;
+  dark: string;
+  darker?: string;
+  accent?: string;
+};
+
+export type FontSpec = {
+  family?: string;
+  size?: string;
+  weight?: string;
+  color?: string;
+};
+
+export type AcademicArea = 'acs' |
+'am' |
+'ap' |
+'be' |
+'cs' |
+'ee' |
+'ese' |
+'general' |
+'mat & me' |
+'mde' |
+'msmba' |
+'sem';
```

Comment:
AcademicArea is specific to the course planner, not sure if they should be in mark-one.

Reply:
Yeah, I've thought this too. Originally the idea was that we'd have the individual area colors as part of Mark One so we can use them across apps, since they're the same colors used on the SEAS website. But it does add the complication of us needed to add any new colors in Mark One before they can be used anywhere else. 

So it's a fair point, but I don't think it should be addressed as part of this ticket since we're just trying to fix a bug a restore functionality. Can you open a new issue on the Course Planner project to address this?

Reply:
Agree, I will open a new ticket issue for this one.